### PR TITLE
S02US10_Footer création d'un footer

### PR DESCRIPTION
**Pull request by maxime-guichard**

### CODE REVIEW

**Review by endricklosange — APPROVED** [review]

File: templates/base/_footer.html.twig [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        </div>
+      <!--Grid row-->
+    </section>
+    <!-- Section: Links -->
+  </div>
+
+  <div class="container p-4">
+    <!-- Section: Social media -->
+    <section class="d-flex justify-content-around mb-4">
+    <!-- Facebook -->
```
Comment (LucyMerlier) [inline templates/base/_footer.html.twig:23]: 2 spaces for indentation => should be 4

File: templates/base/_footer.html.twig [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<!-- Footer -->
+<footer class="bg-secondary text-center text-white">
+
+    <!-- Section: Links -->
+    <section class="">
+      <!--Grid row-->
+      <div class="d-flex justify-content-center row pt-3">
+            <div class="col justify-content-start">
+              <a href="#!" class="text-white text-decoration-none">Demander un devis</a>
```
Comment (LucyMerlier) [inline templates/base/_footer.html.twig:9]: form what I know, suing "#!" is not good practice

File: templates/base/_footer.html.twig [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            </div>
+            <div class="col justify-content-end">
+              <a href="#!" class="text-white text-decoration-none">Nous contacter</a>
```
Comment (LucyMerlier) [inline templates/base/_footer.html.twig:12]: same as above

File: templates/base/_footer.html.twig [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+    </section>
+    <!-- Section: Social media -->
+
+  <!-- Copyright -->
+  <div class="text-center" style="background-color: rgba(0, 0, 0, 0.2);">
+    <p class="text-white">© 2020 Copyright: Student-Interim</p>
+  </div>
+  <!-- Copyright -->
+</footer>
```
Comment (LucyMerlier) [inline templates/base/_footer.html.twig:51]: 2 spaces for indentation => should be 4

File: templates/base/_footer.html.twig [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    </section>
+    <!-- Section: Links -->
+  </div>
+
+  <div class="container p-4">
+    <!-- Section: Social media -->
+    <section class="d-flex justify-content-around mb-4">
+    <!-- Facebook -->
+    <a class="m-1" href="https://www.facebook.com/studentinterim" role="button">
+        <img src="{{ asset('build/images/facebook.png') }}" alt="logo">
```
Comment (LucyMerlier) [inline templates/base/_footer.html.twig:25]: alt="logo" => ok, but logo of what?

File: templates/base/_footer.html.twig [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    </a>
+
+    <!-- Instagram -->
+    <a class="m-1" href="https://www.instagram.com/studentinterim/" role="button">
+        <img src="{{ asset('build/images/insta.png') }}" alt="logo">
```
Comment (LucyMerlier) [inline templates/base/_footer.html.twig:30]: alt="logo" => ok, but logo of what?

File: templates/base/_footer.html.twig [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+    </a>
+
+    <!-- Linkedin -->
+    <a class="m-1" href="https://www.linkedin.com/company/student-int%C3%A9rim" role="button">
+        <img src="{{ asset('build/images/linkedin.png') }}" alt="logo">
```
Comment (LucyMerlier) [inline templates/base/_footer.html.twig:35]: alt="logo" => ok, but logo of what?

File: templates/base/_footer.html.twig [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+    </a>
+
+    <!-- Youtube -->
+    <a class="m-1" href="https://youtube.com/channel/UCqQPYBaPQ7eRUtB-GKWIniQ" role="button"
+        ><img src="{{ asset('build/images/youtube.png') }}" alt="logo">
```
Comment (LucyMerlier) [inline templates/base/_footer.html.twig:40]: alt="logo" => ok, but logo of what?

File: templates/layout.html.twig [hunk 9]
```diff
@@ -1,5 +1,9 @@
...
 {% block navbar %}
-    {% include '_navbar.html.twig' %}
-{% endblock %}
\ No newline at end of file
+	{% include '_navbar.html.twig' %}
+{% endblock %}
+
+{% block footer %}
+	{% include 'base/_footer.html.twig' %}
+{% endblock %}
```
Comment (LucyMerlier) [inline templates/layout.html.twig:9]: if the footer is gonna be present on all pages, why not include it in base.html.twig?

File: templates/layout_company.html.twig [hunk 10]
```diff
@@ -4,4 +4,8 @@
 
 {% block navbar %}
     {% include '_navbar_user.html.twig' %}
+{% endblock %}
+
+{% block footer %}
+	{% include 'base/_footer.html.twig' %}
 {% endblock %}
```
Comment (LucyMerlier) [inline templates/layout_company.html.twig:11]: if the footer is gonna be present on all pages, why not include it in base.html.twig?

File: templates/layout_student.html.twig [hunk 11]
```diff
@@ -4,4 +4,8 @@
 
 {% block navbar %}
     {% include '_navbar_user.html.twig' %}
+{% endblock %}
+
+{% block footer %}
+	{% include 'base/_footer.html.twig' %}
 {% endblock %}
```
Comment (LucyMerlier) [inline templates/layout_student.html.twig:11]: if the footer is gonna be present on all pages, why not include it in base.html.twig?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: templates/layout.html.twig (hunk 9 above)
Comment (maxime-guichard) [reply templates/layout.html.twig:9]: car j'en veux pas sûr ma page admin

**Review by JuMn88 — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**